src/Scenario/StatusWatcherScenario.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TelegramOSINT\Scenario;
+
+use TelegramOSINT\Client\AuthKey\AuthKeyCreator;
+use TelegramOSINT\Client\Client;
+use TelegramOSINT\Client\StatusWatcherClient\Models\HiddenStatus;
+use TelegramOSINT\Client\StatusWatcherClient\Models\ImportResult;
+use TelegramOSINT\Client\StatusWatcherClient\Models\User;
+use TelegramOSINT\Client\StatusWatcherClient\StatusWatcherCallbacks;
+use TelegramOSINT\Client\StatusWatcherClient\StatusWatcherClient;
+use TelegramOSINT\Exception\TGException;
+use TelegramOSINT\LibConfig;
+use TelegramOSINT\Logger\ClientDebugLogger;
+use TelegramOSINT\Logger\Logger;
+use TelegramOSINT\Tools\Proxy;
+
+/**
+ * Status watcher client
+ *
+ * Subscribes to a bunch of accounts and then
+ */
+class StatusWatcherScenario implements StatusWatcherCallbacks, ClientDebugLogger, ScenarioInterface
+{
+    /**
+     * @var StatusWatcherClient
+     */
+    protected $client;
+    /**
+     * @var string
+     */
+    private $authKey;
+    /**
+     * @var Proxy
+     */
+    private $proxy;
+
+    /**
+     * @param ClientGeneratorInterface|null $generator
+     * @param Proxy|null                    $proxy
+     *
+     * @throws TGException
+     */
+    public function __construct(
+        ?ClientGeneratorInterface $generator = null,
+        ?Proxy $proxy = null
+    ) {
+        /*
+         * Set TL-node logger
+         */
+        Logger::setupLogger($this);
+
+        if (!$generator) {
+            $generator = new ClientGenerator(LibConfig::ENV_AUTHKEY);
+        }
+
+        $this->authKey = $generator->getAuthKey();
+
+        /*
+         * Clients init
+         */
+        $this->client = $generator->getStatusWatcherClient($this);
+        $this->proxy = $proxy;
+    }
+
+    /**
+     * @param bool $pollAndTerminate
+     *
+     * @throws TGException
+     */
+    public function startActions(bool $pollAndTerminate = true): void
+    {
+        if ($pollAndTerminate) {
+            $this->monitorPhones();
+        }
+    }
+
+    /**
+     * @throws TGException
+     */
+    private function monitorPhones(): void
+    {
+        $authKey = AuthKeyCreator::createFromString($this->authKey);
+        $this->client->login($authKey, $this->proxy);
+
+        /* add via phone numbers */
+        $monitoringPhones = [
+            '541123683798',
+            '60192003400',
+            '77779479694',
+            '393475723072',
+            '989196190933',
+        ];
```

Comment:
These numbers shoul not be hardcoded

Reply:
right, removed those, now example awaits number list in argv.